docs/Concepts/Permissioning/Onchain-Permissioning.md

```diff
@@ -1,65 +1,75 @@
+remove). There is one list of admin accounts for node and accounts.
 
-* Account Rules - stores the accounts whitelist and account whitelist operations (for example, add and remove). 
+## Permissioning management Dapp
 
-* Admin - stores the list of admin accounts and admin list operations (for example, add and remove). There is 
-one list of admin accounts for node and accounts.
+The [Permissioning Management Dapp] provides view and maintain access to the whitelists.
 
-## Permissioning Management Dapp
+!!! tip
 
-The [Permissioning Management Dapp](../../Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md) is provided to view 
-and maintain the whitelists. 
+    Before v1.2, the management interface used Truffle, now deprecated. For an improved user
+    experience, use the Dapp.
 
-!!! tip 
-    Before v1.2, we provided a management interface using Truffle.
-    The management interface using Truffle is deprecated and we recommend using the Dapp for an improved user experience. 
+### Whitelists
 
-### Whitelists 
+Permissioning implements three whitelists:
 
-Permissioning implements three whitelists: 
+* Accounts, which can submit transactions to the network.
+* Nodes, which can participate in the network.
+* Admins, which are accounts that can update the accounts and nodes whitelists.
 
-* Accounts can submit transactions to the network
+!!! caution "Using account permissioning and privacy"
 
-* Nodes can participate in the network 
+    Account permissioning is incompatible with
+    [random key signing](../../HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md) for
+    [privacy marker transactions](../Privacy/Private-Transaction-Processing.md).
 
-* Admins are accounts that can update the accounts and nodes whitelists 
-
-!!! caution "Using account permissioning and privacy"
-    Account permissioning cannot be used with [random key signing](../../HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md)
-    for [privacy marker transactions](../Privacy/Private-Transaction-Processing.md).
-    
-    If using account permissioning and privacy, a signing key must be specified using the [`--privacy-marker-transaction-signing-key-file`](../../Reference/CLI/CLI-Syntax.md#privacy-marker-transaction-signing-key-file)
-    command line option and the signing key included in the accounts whitelist.  
+    If using account permissioning and privacy, a signing key must be specified using the
+    [`--privacy-marker-transaction-signing-key-file`] command line option and the signing key
+    included in the accounts whitelist.
 
 ## Bootnodes
 
-When a node is added to the network, it connects to the bootnodes until it synchronizes to the chain head regardless of
-node permissions. Once in sync, the permissioning rules in the Account Rules and Node Rules smart contracts are applied.  
+When a node joins a network, it connects to the bootnodes until it synchronizes to the chain head
```

Comment:
I'd check wether the distinction between _When a node is added_ and _When a node joins_ is important with Team Phoenix.  The point of using a permissioned network is to prevent nodes joining.

Reply:
Good point. Updated to "When a network adds a node, the node connects to..."